seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -2647,6 +2647,48 @@
       }
     },
     {
+      "id": "598e8944f009e646fc236146",
+      "title": "Understanding Undefined Value returned from a Function",
+      "description": [
+        "A function can include the <code>return</code> statement but it does not have to. In case that the function does not have a <code>return</code> statement, the function will process the inner code and the returned value in that case will be <code>undefined</code>.",
```

Comment:
I'd tweak this second sentence a little, so it reads: "In the case that the function doesn't have a `return` statement, when you call it, the function processes the inner code but the returned value is `undefined`.